cinema/views.py

```diff
@@ -127,6 +124,28 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "title",
+                type=str,
+                description="Filter by title name (ex, ?=title=Looper)",
```

Comment:
Wrong example in all cases. I never see the querystring like `?=field=value`
correct: `?field=value`